### ISSUE TITLE
Feature: Allow including of SDFs from URL

### ISSUE THREAD

**Issue by jbpratt** [RedHatQE/teflo#130]
```
---
name: ...
description: ...

include:
- orchestrate.yml
- https://github.com/RedHatQE/teflo/examples/execute.yml
```

This should resolve any includes the remote SDF may have as well.

**Comment by dno-github** [comment]
Jira ticket CCITCARBON-368 has been created for this issue. Url : https://issues.redhat.com/browse/CCITCARBON-368

**Comment by rujutashinde** [comment]
fixed with #168
